### PR TITLE
LINK-1294 | Group enrolment DB structure

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Adds a new model `SignUpGroup` for keeping track which `SignUp` instances belong to the same group and who the people (= sign-ups) responsible for the group are. API functionality will be added later for `SignUpGroup` in other tasks.
### Closes
[LINK-1294](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1294)

[LINK-1294]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1294?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jorilindell** [comment]
In design https://www.figma.com/file/OVZdkK7BZgG0u30PGIc785/%F0%9F%94%97-LinkedEvents?type=design&node-id=1%3A4499&mode=design&t=aIkvdQODKYaTwvCp-1 there is additional info fields for each signups and separate one for the group. For each signup it's named as "Lisätietoja (valinnainen)" and for the group it's names as "Lisätietoa ilmoittautumisesta (valinnainen)". 

At the moment there is an `extra_info` field in SignUp model, but UI still needs a field to store "Lisätietoa ilmoittautumisesta (valinnainen)" info.

We can also create a separate JIRA ticket if this is out of the scope.

**Comment by harriris-vincit** [comment]
@jorilindell thanks for pointing that field out - I had missed the requirement. I will add the field extra info field to the group tomorrow in this PR.

File: registrations/models.py
Comment (voneiden) [inline registrations/models.py:281]: Sonarcloud complains about having a nullable text field, but it's pretty much in line with the existing code base so I don't see a need to change it. If no changes will be made then feel free to close the sonarcloud issue.

**Review by voneiden — COMMENTED** [review]
Looks OK, before approving I can double check still after #675 is merged into this.

**Review by jorilindell — APPROVED** [review]
Looks good to me

**Review by voneiden — APPROVED** [review]
Approved as merge strategy has been clarified

**Pull request merged**
